File: tourneyAPI/Models/Entities/Characters/Bowser.cs
namespace Entities;

using Enums;

// Defines the competitive profile metadata for this playable character.
public class Bowser : Character
{
    // Initializes this character's default competitive attributes for matchmaking and tier logic.
    public Bowser()
    {
        Id = CharacterId.Bowser;
        characterName = CharacterName.BOWSER;
        archetype = Archetype.BAIT_AND_PUNISH;
        fallSpeed = FallSpeed.FAST_FALLERS;
        weightClass = WeightClass.SUPER_HEAVYWEIGHT;
        tierPlacement = TierPlacement.C;
    }

}

File: tourneyClient/src/models/entities/Characters/Bowser.ts
import { CharacterId } from "@/models/Enums/CharacterId";
import { Character } from "../Character"
import { CharacterName } from "@/models/Enums/CharacterName";
import { Archetype } from "@/models/Enums/Archetype";
import { FallSpeed } from "@/models/Enums/FallSpeed";
import { WeightClass } from "@/models/Enums/WeightClass";
import { TierPlacement } from "@/models/Enums/TierPlacement";

export const Bowser: Character =
{
    id: CharacterId.Bowser,
    characterName: CharacterName.BOWSER,
    archetype: Archetype.BAIT_AND_PUNISH,
    fallSpeed: FallSpeed.FAST_FALLERS,
    weightClass: WeightClass.SUPER_HEAVYWEIGHT,
    tierPlacement: TierPlacement.C
}

export default Bowser; 